custom_components/tahoma/config_flow.py

```diff
@@ -49,3 +49,20 @@ async def async_step_user(self, user_input=None):
         return self.async_show_form(
             step_id="user", data_schema=DATA_SCHEMA, errors=errors
         )
+
+    async def async_step_import(self, import_info):
```

Comment:
import_info is the usual name?

Reply:
I changed it to `import_config` which is also commonly used, however I like that one more. :-)